stacker_blueprints/empire/policies.py

```diff
@@ -117,7 +178,35 @@ def logstream_policy():
                     kinesis.CreateStream, kinesis.DescribeStream,
                     Action(kinesis.prefix, "AddTagsToStream"),
                     Action(kinesis.prefix, "PutRecords")
-                ])
-        ]
-    )
+                ])])
+    return p
+
+
+def runlogs_policy(log_group_ref):
```

Comment:
I don't know why we originally made this a separate function in our internal stuff. This should/cloud just be in the empire_policy.